src/common/app.config.js

```diff
@@ -535,6 +536,24 @@ const appConfig = /* @ngInject */ function (envServiceProvider, $compileProvider
     PREFERRED_RADIO_STATION: 'Preferred Radio Station: '
   })
   $translateProvider.preferredLanguage('en')
+
+  window.datadogRum = datadogRum
+  window.datadogRum && window.datadogRum.init({
+    applicationId: '3937053e-386b-4b5b-ab4a-c83217d2f953',
+    clientToken: process.env.DATADOG_RUM_CLIENT_TOKEN,
```

Comment:
@dr-bizz Is this still the correct way to get the client token now that it is on terraform?

Reply:
I think you also need to add the variable here: webpack.config.js on line 58. So the front end can read it.

```
DATADOG_RUM_CLIENT_TOKEN: process.env. DATADOG_RUM_CLIENT_TOKEN || ''

Reply:
You'll also need it in `build.yml` for GHA I think.